ipu-plugin/pkg/ipuplugin/deviceplugin.go

```diff
@@ -8,27 +8,30 @@ import (
 	"slices"
 	"strings"
 
+	"github.com/intel/ipu-opi-plugins/ipu-plugin/pkg/types"
 	pb "github.com/openshift/dpu-operator/dpu-api/gen"
 	pluginapi "k8s.io/kubelet/pkg/apis/deviceplugin/v1beta1"
 )
 
 type DevicePluginService struct {
 	pb.UnimplementedDeviceServiceServer
+	mode string
 }
 
 var (
 	exclude     = []string{"enp0s1f0", "enp0s1f0d1", "enp0s1f0d2", "enp0s1f0d3"}
 	sysClassNet = "/sys/class/net"
 	deviceCode  = "0x1452"
+	deviceCodeVf = "0x145c"
```

Comment:
I think we need a list of device codes that have better names:
deviceCode => these are APF interfaces right? maybe deviceCodeDpuAPF
deviceCodeVf => these are the host VFs. Maybe deviceCodeHostVf

Reply:
some of this is existing code. There are  other changes in this file expected, can look into changing at that time.